homework-2/src/components/shared/SidebarNavigation/SidebarNavigation.tsx

```diff
@@ -2,16 +2,36 @@
 
 import { Flex, Heading, Text } from "@chakra-ui/react"
 import NextLink from 'next/link';
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const handleLogout = () => {
+        localStorage.removeItem('authorization-header');
+        window.location.href = '/login';
+}
 
 export const SidebarNavigation = () => {
+    const [loggedIn, setLoggedIn] = useState(false);
+
+    useEffect(() => {
+        const auth = localStorage.getItem('authorization-header');
+        
+        if(auth){
+            setLoggedIn(true);
+        }
+    }, [])
+    
     return (
         <Flex as='nav' flexDirection='column' backgroundColor='#280454' color='white' height='100vh' width='10vw' position='fixed' top={0} left={0} p={30} gap={10}>
             <Heading as='h2' size='xl' mb={50} >TV SHOWS APP</Heading>
 
             <Text as={NextLink} href={`/all-shows`}>All shows</Text>
-            <Text as={NextLink} href={`/top-rated`} >Top rated</Text>
-            <Text as={NextLink} href={`/my-profile`} >My profile</Text>
-            <Text as={NextLink} href={`/log-out`} mt='auto' >Log out</Text>
+            <Text as={NextLink} href={`/top-rated`}>Top rated</Text>
+            <Text as={NextLink} href={`/my-profile`}>My profile</Text>
+
+            {!loggedIn && <Text as={NextLink} href={`/register`} mt='auto'>Register</Text>}
+            {!loggedIn && <Text as={NextLink} href={`/login`}>Log in</Text>}
+            {loggedIn && <Text as={NextLink} href={`/log-out`} onClick={handleLogout} mt='auto'>Log out</Text>}

```

Comment:
Nakon što se ulogiram, u sidebaru su mi i dalje vidljivi Login i iRegister linkovi. Maknu se tek nakon refresha